CHANGES

```diff
@@ -0,0 +1,11 @@
+0.1 (unreleased)
+----------------
+
+- Make shaders directory a python package and add included shader file paths to __init__.py
+
+
+0.0.1.dev0 (2020-05-23)
+-----------------------
+
+- Initial release
+
```

Comment:
Just curious, why not a `CHANGELOG.md` instead?
https://github.com/olivierlacan/keep-a-changelog/blob/v1.1.0/CHANGELOG.md

Reply:
habitual. no specific reason